glommio/src/task/header.rs

```diff
@@ -27,6 +27,9 @@ pub(crate) struct Header {
     /// Current state of the task.
     pub(crate) state: u8,
 
+    /// Latency matters or not
+    pub(crate) latency_matters: bool,
```

Comment:
may be better to represent it as `Option<RawFd>`, and add `Some(eventfd)`. `None` would signal that we don't need to write to the eventfd.